pkg/image/controller/scheduled_image_controller.go

```diff
@@ -189,6 +190,13 @@ func resetScheduledTags(stream *imagev1.ImageStream) {
 func needsScheduling(stream *imagev1.ImageStream) bool {
 	for _, tagRef := range stream.Spec.Tags {
 		if tagImportable(tagRef) && tagRef.ImportPolicy.Scheduled {
+			if ref, err := imageref.Parse(tagRef.From.Name); err != nil && len(tagRef.From.Name) > 0 {
+				return false
```

Comment:
shouldn't the validation code prevent this from happening?  (it's the same check the validation is doing, right?)

Reply:
i guess if you have to parse it anyway to get the id it makes sense to respect the error, if any though.

Reply:
that is where I landed on this one as well